packages/blade/src/components/Modal/Modal.web.tsx

```diff
@@ -31,41 +32,52 @@ import { componentZIndices } from '~utils/componentZIndices';
 import { useVerifyAllowedChildren } from '~utils/useVerifyAllowedChildren';
 import { makeAnalyticsAttribute } from '~utils/makeAnalyticsAttribute';
 
-const entry = keyframes`
-  from {
-    opacity: 0;
-    transform: translate(-50%, -50%) scale(0.9) translateY(20px);
-  }
-  to {
-    opacity: 1;
-    transform: translate(-50%, -50%) scale(1) translateY(0px);
-  }
-`;
+const ModalContent = styled(BaseBox)<{ isVisible: boolean; size: NonNullable<ModalProps['size']> }>(
+  ({ isVisible, theme, size }) => {
+    const centerTransform = size === 'full' ? 'none' : 'translate(-50%, -50%)';
```

Comment:
Is this animation defined from design?

Reply:
nope. i have kept is same for normal and full size modal. 

https://github.com/user-attachments/assets/d93cb4d9-7e00-45f5-81d2-c877c708b801

but we don't need translate(-50%, -50%) .. in case of full page modal. since it will already at center. 
also had a discussion with RK. this animation looks weird for full page modal will update it to fade in and fade out